examples/fcis/mxnet2npz.py

```diff
@@ -9,7 +9,7 @@ def main():
     parser = argparse.ArgumentParser(
         description='Script to convert mxnet params to chainer npz')
     parser.add_argument(
-        'mxnet-param-file', help='Mxnet param file i.e. fcis_coco-0000.params')
+        'mxnet_param_file', help='Mxnet param file i.e. fcis_coco-0000.params')
```

Comment:
Can you not change this part?
The command line arguments in ChainerCV do not use `_`.

Reply:
Thx, updated. I use `metavar` to show `mxnet-param-file` for command line help.

```
$ python mxnet2npz.py --help
usage: mxnet2npz.py [-h] [--process] [--dataset {sbd,coco}] [--out OUT]
                    mxnet-param-file

Script to convert mxnet params to chainer npz

positional arguments:
  mxnet-param-file      Mxnet param file i.e. fcis_coco-0000.params

optional arguments:
  -h, --help            show this help message and exit
  --process
  --dataset {sbd,coco}
  --out OUT, -o OUT
```